flood_adapt/object_model/hazard/hazard.py

```diff
@@ -663,6 +672,105 @@ def postprocess_sfincs(self):
+                fig = px.line(
+                    df[col] * conversion_factor
+                    + (
+                        self.site.attrs.water_level.localdatum.height.convert(gui_units)
+                        - self.site.attrs.water_level.msl.height.convert(gui_units)
+                    ),  # convert model output to MSL
+                    +self.site.attrs.water_level.msl.height.convert(gui_units),
+                )
+
+                # plot reference water levels
+                fig.add_hline(
+                    y=self.site.attrs.water_level.msl.height.convert(gui_units),
+                    line_dash="dash",
+                    line_color="#000000",
+                    annotation_text="MSL",
+                    annotation_position="bottom right",
+                )
+                if self.site.attrs.water_level.other:
+                    for wl_ref in self.site.attrs.water_level.other:
+                        fig.add_hline(
+                            y=wl_ref.height.convert(gui_units),
+                            line_dash="dash",
+                            line_color="#3ec97c",
+                            annotation_text=wl_ref.name,
+                            annotation_position="bottom right",
+                        )
+
+                fig.update_layout(
+                    autosize=False,
+                    height=100 * 2,
+                    width=280 * 2,
+                    margin={"r": 0, "l": 0, "b": 0, "t": 20},
+                    font={"size": 10, "color": "black", "family": "Arial"},
+                    title={
+                        "text": gdf.iloc[ii]["Description"],
+                        "font": {"size": 12, "color": "black", "family": "Arial"},
+                        "x": 0.5,
+                        "xanchor": "center",
+                    },
+                    xaxis_title="Time",
+                    yaxis_title=f"Water level [{gui_units}]",
+                    yaxis_title_font={"size": 10, "color": "black", "family": "Arial"},
+                    xaxis_title_font={"size": 10, "color": "black", "family": "Arial"},
+                    showlegend=False,
+                )
+
+                # check if event is historic
+                if self.event.attrs.timing == "historical":
+                    # check if observation station has a tide gauge ID
+                    # if yes to both download tide gauge data and add to plot
+                    if isinstance(self.site.attrs.obs_point[ii].ID, int):
+                        df_gauge = HistoricalNearshore.download_wl_data(
+                            station_id=self.site.attrs.obs_point[ii].ID,
+                            start_time_str=self.event.attrs.time.start_time,
+                            stop_time_str=self.event.attrs.time.end_time,
+                            units=UnitTypesLength(gui_units),
+                        )
+                        import plotly.graph_objects as go
```

Comment:
If it doesn't interfere with other imports (which it shouldn't), just put them on top